animalfacts.py

```diff
@@ -1482,6 +1482,7 @@ def animalfactsbot(reddit):
     'Honey badgers are nocturnal',
     'Honey badgers are mostly solitary and only meet up to hunt or mate',
     'Beekeepers will often keep their bees elevated to prevent honey badgers from getting to their hives.',
+    'Honey badgers are substantially resistant to venom, and easily survive snakebites that would kill a human'
```

Comment:
Can you put a comma on the end here for consistency? Otherwise looks good!

Reply:
Done!